bindings/rust/standard/Cargo.toml

```diff
@@ -0,0 +1,5 @@
+[workspace]
+members = [
```

Comment:
Should bench be included here?

Reply:
Also I think I'd prefer two "sibling" workspaces

- `extended`
- `standard`

I think this current approach might be a bit confusing because we end up with a single folder that contains
- crates in a workspace (s2n-tls, s2n-tls-sys)
- crates not in a workspace (bench, generate)
- a workspace (standard)

And 3 seems much more complicated than 2.

Reply:
And it would be nice to have a README explaining _why_ we have the two different workspaces.